solutions/control flow/train.js

```diff
@@ -0,0 +1,21 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+let totalCarsAndCarNumberFromFront = readLine().split(" ");
```

Comment:
Extremely long variable names are counter-productive to readability. Please rename this to something like `inputs`.